public/js/components/RightSidebar.js

```diff
@@ -80,6 +80,11 @@ const RightSidebar = React.createClass({
 
   setupKeyboardShortcuts() {
     const { keyShortcuts } = this.props;
+    if (this.keyShortcutsEnabled) {
+      return;
+    }
+
+    this.keyShortcutsEnabled = true;
```

Comment:
I don't think you'd need this if you used `componentDidMount`, instead of `componentDidUpdate`. Typically you set up this kind of stuff when it mounts and then remove it when it unmounts.